deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,59 @@
+{
+  'variables': {
+    'arm_fpu%': '',
+    'target_arch%': '',
+  },
+
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'direct_dependent_settings': {
+        'include_dirs': ['.'],
+      },
+      'sources': ['simdutf.cpp'],
+      'conditions': [
+        [ 'arm_fpu=="neon" and target_arch=="arm"', {
+          'conditions': [
+            [ 'OS!="win"', {
+              'cflags': [ '-mfpu=neon' ],
+              'xcode_settings': {
+                'OTHER_CFLAGS': [ '-mfpu=neon' ]
+              },
+            }],
+          ],
+        }],
+
+        # Runtime detection will happen for x86 CPUs
+        [ 'target_arch in "ia32 x64 x32"', {
+          'conditions': [
+            [ 'OS!="win"', {
+              'cflags': [
+                '-msse4.2',
+                '-mavx',
+                '-mavx2',
```

Comment:
I'm confused. The comment says it does runtime detection but you still pass -mavx and -mavx2? That means it won't run on non-AVX machines.

Aside: you can drop x32, no longer supported.

Reply:
I think my comment is wrong. What is the correct configuration for not breaking non-avx supported architectures?

Reply:
Unfortunately, builds still fail on Linux.